CHANGELOG.md

```diff
@@ -3,6 +3,62 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.11.2
+
+_Aug 16, 2023_
+
+We'd like to offer a big thanks to the 7 contributors who made this release possible. Here are some highlights ✨:
+
+- 🏎️ Lower the filtering delay in the grid
+- 🌍 Improve Spanish (es-ES) locale on the data grid
+- 🐞 Bugfixes
+- 📚 Documentation improvements
+
+
+### Data Grid
+
+#### `@mui/x-data-grid@v6.11.2`
+
+- [DataGrid] Docs: update column types (#10040) @romgrk
+- [DataGrid] Fix eval blocked by CSP (#9863) @romgrk
+- [DataGrid] Fix row id bug (#10051) @romgrk
+- [DataGrid] Lower filter debounce delay (#9712) @romgrk
```

Comment:
Since we change the default delay here, do you think it should be released in a minor version instead of the patch?
My main concern is that in the case of server-side filtering, requests might be sent too often

Reply:
I felt like that change was a performance optimization, not a new feature. I generally don't obsess too much over minor vs patch if it's unclear, so I'd leave as is, but I can also change it.

Reply:
Alright, I'm fine with patch 👍